js/web/lib/wasm/jsep/webgpu/types.ts

```diff
@@ -15,6 +15,16 @@ export enum GpuDataType {
 }
 export type GpuDataId = number;
 
+export type GpuArchitecture = 'ampere';
+export type GpuVendor = 'amd'|'intel'|'nvidia';
+export interface AdapterInfo {
+  readonly architecture: string;
+  readonly vendor: string;
```

Comment:
It looks like these 2 properties are not needed in the interface. otherwise, users may not necessarily use `isArchitecture()` and `isVendor()`.

Reply:
done